tuist/tuist
# Move lockfile linting from Fourier to a bash script

### Short description 📝
Continuing with our migration out of Fourier, I'm moving the script that ensures the lock files of the Swift Package and the Tuist project match. As part of that work, I noticed that the script was not working as expected because the schema of the JSON had changed, and the Ruby logic became noop.

### How to test the changes locally 🧐
Run `make lint/lockfiles`. If you want to make the task fail, edit the `Package.resolved` at the root and revise one of the packages. The task should fail

### Contributor checklist ✅


### Reviewer checklist ✅
